lib/components/Widgets/Charts/ChartContainer.tsx

```diff
@@ -20,18 +19,18 @@ const Container = forwardRef<
             <div className="mb-0.5 text-sm text-muted-foreground">
               {summary.label}
             </div>
-            <div className="flex flex-row items-end gap-0.5">
-              <ChartCounter value={summary.value} />
```

Comment:
Why was this needed? if it is because of decimals, we can probably adjust the ChartCounter component.
Not sure about the unit being a prefix instead of a suffix. Maybe we need both cases?

Reply:
[fix chart counter decimal points and add post and prefix units to cha…](https://github.com/factorialco/factorial-one/pull/449/commits/fd1c0f890c4f58327ef8c7c17ad55932c69d3f26)